packages/console/src/components/Select/index.module.scss

```diff
@@ -36,24 +35,19 @@
     }
   }
 
-  .clear {
+  .icon {
     position: absolute;
     right: _.unit(2);
-    top: 6px;
-    display: none;
-    color: var(--color-icon);
-  }
 
-  .arrow {
-    position: absolute;
```

Comment:
Want to confirm that we don't need this anymore?

Reply:
because the `.arrow` and `.clear` have the same position logic, I extract them into the `.icon` class.